.npmignore

```diff
@@ -0,0 +1,2 @@
+.travis.yml
+.npmignore
```

Comment:
Hmm, no, it is not ignoring it by default. I tested and without this line it will get also the `.npmignore` file.